src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input.ts

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled';
+import * as C from './colors';
+import * as F from './fonts';
+
+export const InputField1 = styled.input`
+  height: 70px;
+  width: 300px;
+  background-color: ${C.GreyLight};
+  padding-left: 30px;
+  color: ${C.GreySecondaryText};
+  font-weight: ${F.SemiBold};
+  font-size: ${F.XL};
+  border: none;
+  border-radius: 17px;
+
+  ::placeholder,
+  ::-webkit-input-placeholder {
```

Comment:
Please remove ::-webkit-input-placeholder, ::-ms-input-placeholder,  `::placeholder` is fine ` vendor prefixes` are automatically added in production.

Reply:
Done!

Reply:
Thanks just left another review :)